src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -155,6 +155,9 @@ private void InitFamilyInstance(Autodesk.Revit.DB.FamilySymbol fs, XYZ pos)
 
         private void InternalSetLevel(Autodesk.Revit.DB.Level level)
         {
+            if (InternalFamilyInstance.LevelId.Compare(level.Id) == 0)
```

Comment:
The first time is when the family instance is created, the level will be set there directly.